newIDE/app/src/Utils/Analytics/EventSender.js

```diff
@@ -352,24 +366,27 @@ export const sendSubscriptionDialogShown = () => {
   recordEvent('subscription-dialog-shown', {});
 };
 
-export const sendAssetOpened = ({
-  id,
-  name,
-}: {|
+export const sendAssetOpened = (options: {|
   id: string,
   name: string,
+  assetPackName: string | null,
+  assetPackTag: string | null,
+  assetPackId: string | null,
+  assetPackKind: 'public' | 'private' | 'unknown',
 |}) => {
-  recordEvent('asset-opened', { id, name });
+  recordEvent('asset-opened', options);
 };
 
-export const sendAssetAddedToProject = ({
-  id,
-  name,
-}: {|
+export const sendAssetAddedToProject = (options: {|
   id: string,
   name: string,
+  kind: 'public' | 'private' | 'unknown',
```

Comment:
is there value in adding this prop?
An asset will probably always have the same kind as its pack

Reply:
Probably yes - but this is more for on the posthog/analytics side, to be able to explore multiple events and do a breakdown (i.e: a "group by" in SQL, a "breakdown" in Posthog) using the same field for multiple events.

Reply:
This being said, I was assuming other events where having this "kind" for asset, which is not the case. I'm removing it!